tensorboard/uploader/uploader_main.py

```diff
@@ -367,9 +367,15 @@ def execute(self, channel):
       msg = 'Output directory already exists: %r' % outdir
       raise base_plugin.FlagsError(msg)
     num_experiments = 0
-    for experiment_id in exporter.export():
-      num_experiments += 1
-      print('Downloaded experiment %s' % experiment_id)
+    try:
+      for experiment_id in exporter.export():
+        num_experiments += 1
+        print('Downloaded experiment %s' % experiment_id)
+    except exporter_lib.NewExceptionType as e:
+      print(
+        '\nUploader has failed because of an internal error.  Please reach '
```

Comment:
Do you think that we should give a more specific error message here, as
we know that it’s a timeout and not just an arbitrary internal error?

Reply:
Done.